eternalcore-core/src/main/java/com/eternalcode/core/translation/Translation.java

```diff
@@ -101,6 +101,12 @@ interface InventorySection {
         String disposalTitle();
     }
 
+    interface SeenSection {
+        Notice neverPlayedBefore();
+        Notice lastSeen();
+        Notice nowOnline();
+    }
```

Comment:
Click F6 and use this structure:

Also for interface implementation.

![image](https://github.com/user-attachments/assets/70d692c8-aa2e-4f89-b7e0-0f402e73dea9)